cmd/soroban-cli/src/contract/deploy.rs

```diff
@@ -114,11 +119,15 @@ impl Cmd {
                 })?,
             );
 
-        if self.config.is_no_network() {
+        let contract_id = if self.config.is_no_network() {
             self.run_in_sandbox(hash)
         } else {
             self.run_against_rpc_server(hash).await
-        }
+        };
+
+        // Add sleep so horizon can ingest new account state
+        sleep(Duration::from_secs(2)).await;
+        contract_id
```

Comment:
Can you add a comment indicating this should be removed once `getAccount` is removed?